### ISSUE TITLE
this._helpers.createEntityRow is not a function

### ISSUE THREAD

**Issue by Mariusthvdb** [iantrich/restriction-card#15]
**Checklist:**

- [x] I updated to the latest version available
- [x] I cleared the cache of my browser

**Release with the issue:**
1.1.8
**Last working release (if known):**
not sure
**Browser and Operating System:**
Safari Mac
Hassio 106.5

**Description of problem:**

card config:
```
      - type: custom:restriction-card
        card:
          entity: input_boolean.everspring_no
        restrictions:
          block: true
        row: true
```
**Javascript errors shown in the web inspector (if applicable):**

```
this._helpers.createEntityRow is not a function
```

**Additional information:**
<img width="500" alt="Schermafbeelding 2020-03-07 om 11 39 15" src="https://user-images.githubusercontent.com/33354141/76141858-4f359b80-6068-11ea-867c-eb7192fd0b81.png">

<img width="734" alt="Schermafbeelding 2020-03-07 om 11 41 52" src="https://user-images.githubusercontent.com/33354141/76141903-b6535000-6068-11ea-9651-0b4c6995e2c5.png">

Chrome:
<img width="546" alt="Schermafbeelding 2020-03-07 om 11 42 59" src="https://user-images.githubusercontent.com/33354141/76141930-d7b43c00-6068-11ea-9d7d-9ad1f563de80.png">

Safari:

<img width="412" alt="Schermafbeelding 2020-03-07 om 11 43 34" src="https://user-images.githubusercontent.com/33354141/76141944-e569c180-6068-11ea-81ed-f5431fb84b5a.png">

**Comment by ricreis394** [comment]
I'm having the same error
![restriction bug](https://user-images.githubusercontent.com/11768109/76413339-26294980-638d-11ea-8f1e-273f037889b1.jpg)

**Comment by iantrich** [comment]
should be fixed in 1.1.9
